client/assets/langs/en.json

```diff
@@ -106,11 +106,12 @@
   "UNBAN_USER": "Unban User",
   "UNBAN_USER_TEXT": "Do you wan to unban current user? If then, you will start to see and receive messages from the user.",
   "UPDATE": "Update",
+  "UPDATE_PROFILE": "Update profile",
```

Comment:
How about "Profile is updated" or "Profile update complete"?

Reply:
@0916dhkim  Thank you for your advice!
I will update to "Profile update complete". 😄